easybuild/easyconfigs/s/Salome/Salome-8.5.0-intel-2020a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Salome'
+version = '8.5.0'
+
+easyblock = "Tarball"
+
+homepage = 'https://www.salome-platform.org/'
+description = """Salome is an open-source software that provides a generic Pre- and Post-Processing platform for numerical
+ simulation. It is based on an open and flexible architecture made of reusable components."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
```

Comment:
It seems that Salome has been built with GCC so using the `intel` toolchain is not necessary. Since it needs FFTW, we could use the `foss` toolchain.